### DISCUSSION TITLE
numpy ~= 1.26.0 creates dependency conflicts with other packages

### DISCUSSION THREAD

**Discussion by ColtonBehannon** [microsoft/semantic-kernel#8673] [Q&A] [answered]
When creating to create a Docker image of my application that utilizes the SK Python SDK I'm getting a dependency conflict with a few other packages such as [chainlit](https://github.com/Chainlit/chainlit) and [STORM](https://github.com/stanford-oval/storm). These require numpy >=1.26.0 whereas SK requires ~= 1.25.0 for Python versions less than 3.12. I was wondering what the reason behind this was and if there was room to loosen this requirement? If not, does anyone have suggestions to get around this?

**Comment by moonbox3** [comment, marked as answer]
Hi @ColtonBehannon, it looks like numpy 1.26.0 supports Python 3.10 (and up). I'm not sure why there's a distinction in our code -- perhaps to ensure backwards compat. I'll work on this change in the next day or so.

**Comment by moonbox3** [comment]
Config change is here: https://github.com/microsoft/semantic-kernel/pull/8819

**Comment by moonbox3** [comment]
@ColtonBehannon SK Python 1.9.0 with this config change is now available.

**Reply by ColtonBehannon** [reply]
Awesome, thanks for the quick response and the update!
